torch/_torch_docs.py

```diff
@@ -3571,6 +3571,34 @@ def merge_dicts(*dicts):
     tensor([15, 30, 15])
 """.format(**common_args))
 
+add_docstr(torch.ldexp, r"""
+ldexp(input, other, *, out=None) -> Tensor
+
+Multiplies :attr:`input` by 2**:attr:`other`.
+
+.. math::
+    \text{{out}}_i = \text{{input}}_i * 2^\text{{other}}_i
+""" + r"""
+
+Typically :attr:`input` contains mantissas to be multiplied by the specified specified
+integral powers of two in :attr:`other`, in order to construct floating point numbers.
+
+Args:
+    {input}
+    other (Tensor): a Tensor containing integral types.
```

Comment:
Now that ldexp is a composite operation there's no requirement that other be an integer tensor. You could say, "a tensor of exponents, typically integers". Note that "tensor" is lower case when we're referring to it in a descriptive context. It is still uppercased when describing the type of the argument, however.